website_sale_recently_viewed_products/README.rst

```diff
@@ -0,0 +1,50 @@
+========================
```

Comment:
But by giving it the same "decoration" style as the other sections, they all get the same semantic meaning. Try for example converting to latex or html using `rst2latex` or `rst2html`, all the sections get `\section` (LaTeX) or `h1` (HTML) elements, while in my way the module name gets `\title` or `h1` while the various parts get `\section` or `h2`